network/src/protocols/discovery/test.rs

```diff
@@ -317,9 +338,91 @@ fn outbound() {
         let msg = recv_msg(listener_substream).await.unwrap();
         assert_eq!(1, msg.get_notes().len());
         assert_eq!(Vec::from(peer_id), msg.get_notes()[0].get_peer_id());
-        assert_eq!(address, get_addrs(&msg.get_notes()[0])[0]);
+        assert_eq!(addrs, get_addrs_from_note(&msg.get_notes()[0]));
     };
 
     rt.block_on(f_peer_mgr.boxed().unit_error().compat())
         .unwrap();
 }
+
+#[test]
+fn addr_update_includes_seed_addrs() {
+    ::logger::try_init_for_testing();
+    let mut rt = Runtime::new().unwrap();
+    // Setup self.
```

Comment:
maybe insert a new line before this to improve readability

Reply:
Ahh, I was mostly copying @bothra90 's style of the surround tests. But I agree; I prefer more new lines.